Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp

```diff
@@ -1535,7 +1535,7 @@ static bool encodeArgument(IPC::Encoder& encoder, JSIPC& jsIPC, JSContextRef con
     }
 
     if (type == "Vector") {
-        if (!jsValue.isObject() || !jsValue.inherits<JSC::JSArray>(vm)) {
+        if (!jsValue.isObject() || !jsValue.inherits<JSC::JSArray>()) {
```

Comment:
Do we really need the isObject check here?

Reply:
We do not need it. Dropped.